core/src/main/resources/hudson/PluginManager/advanced.jelly

```diff
@@ -59,12 +59,12 @@ THE SOFTWARE.
                                 ${%deploytext}
                             </div>
                             <f:entry title="${%File}">
-                                <input type="file" name="name" class="setting-input" accept=".hpi,.jpi"/>
+                                <input type="file" name="name" class="jenkins-input" accept=".hpi,.jpi"/>
```

Comment:
seems broken?

When I click this in chrome nothing happens

Reply:
Can't seem to reproduce this in Safari or Chrome